content/docs/tutorials/certificate-defaults/README.md

```diff
@@ -0,0 +1,502 @@
+---
+title: Setting default certificate values
+description: |
+  Explore how we can use open-source solutions to provide default
+  values to Certificate resources without enforcing them.
```

Comment:
Here's how this looks when someone shares the link:
 
<img width="607" alt="image" src="https://github.com/cert-manager/website/assets/978965/2a72c396-d0cf-4bd7-870a-b0e6613ed1f1">


I suggest changing the title / description to:
> Learn how to set best-practice Certificate defaults automatically
> 
> Learn how to use Kyverno ClusterPolicy to automatically set best-practice default  field values on all cert-manager Certificate resources.

...or something like that.

Reply:
I edited based on your thoughts. See what you think of attempt no. 2